web/src/pages/profile.tsx

```diff
@@ -535,6 +536,8 @@ const ProfilePage = (props: Props): ReactElement => {
         />
         <hr className="margin-top-4 margin-bottom-2" aria-hidden={true} />
         <FieldLabelProfile fieldName="taxId" />
+        <br />
+        <Content>{Config.profileDefaults.fields.taxId.default.disclaimerMd}</Content>
```

Comment:
In this and all the places where it changed, there is missing logic here. The disclaimer should only show for Trade Name businesses